### ISSUE TITLE
Build fails when browserslist is configured using Different Environments

### ISSUE THREAD

**Issue by zsobral** [doczjs/docz#1031]
## Bug Report

**Build fails**

when I use this config in the `package.json` the build fails

```json
"browserslist": {
  "production": [
    ">0.2%",
    "not dead",
    "not op_mini all"
  ],
  "development": [
    "last 1 chrome version",
    "last 1 firefox version",
    "last 1 safari version"
  ]
}
```

```
Unknown error from PostCSS plugin. Your current PostCSS version is 6.0.23, but autoprefixer uses 7.0.17. Perhaps this is the source of 
the error below.
 ERROR  Failed to compile with 1 errors                                                                                        19:22:33
⠀
 error  in ./node_modules/gatsby-theme-docz/src/index.css
⠀
Module build failed (from ./node_modules/gatsby/node_modules/postcss-loader/lib/index.js):
BrowserslistError: Browser queries must be an array or string. Got object.
```

**Expected behavior**

expected to build successful since is a valid browserconfig

**Environment**

* OSX 10.14.6
* Node v12.8.0
* Yarn 1.17.3
* docz 2.0.0-rc.5

**Comment by rakannimer** [comment]
Hello @zsobral 

Thank you for reporting this. This issue was fixed starting from docz `2.0.0-rc.33`

If you try updating your setup it should work as expected : 

```sh
rm -rf .docz node_modules yarn.lock package-lock.json && yarn && yarn dev
```

**Comment by zsobral** [comment]
@rakannimer thanks, it's working now
